gui/system/forms.py

```diff
@@ -2853,6 +2853,59 @@ class Meta:
         model = models.CertificateAuthority
 
 
+class CertificateAuthoritySignCSRForm(ModelForm):
+    cert_CSRs = forms.ChoiceField(
```

Comment:
Perhaps you can use ModelChoiceField field here, then you wont need to set the choices by yourself in `__init__`